packages/checkbox-group/test/lion-checkbox-group.test.js

```diff
@@ -25,4 +25,66 @@ describe('<lion-checkbox-group>', () => {
     el.formElements['sports[]'][0].checked = true;
     expect(el.hasFeedbackFor).to.deep.equal([]);
   });
+
+  it('is accessible', async () => {
+    const el = await fixture(html`
+      <lion-radio-group name="dinosGroup" label="What are your favourite dinosaurs?">
```

Comment:
This is the test file for checkboxes, not radios.

Reply:
Best copy/paster 2019